WaysofWork.md

```diff
@@ -1,169 +1,178 @@
 # onBOARDing document: Issues, Milestones and Projects for the DSH team
-V: Beta
+V: 1
 This document lays out how to create and use this repo and associated project board for the tracking and organisation of work relating to DSH project, attending to its multi-repository nature.
 
 The proposed structure responds to an ask from the project team to increase communication, organization and collaboration with the least possible amount of meeting time or fixed synchronous co-working time.
 
-This is a first proposal so please do propose changes via PR.
+These Ways of Work are inspired by Agile principles and SCRUM methodology, although readers may recognize very little from the latter in the resulting practices. The main goals are to increase communication, ensure team members are directly involved in deciding how to carry out the project, protect their workloads and improve visibility of the project's impact.
+
+They have been developed with the team and attending to the meeting-intense research environment of the project, this made it challenging to introduce SCRUM ceremonies or make it mandatory for all team members to take part in each one. As a result asynchronous reporting have been prioritised and some ceremonies blended together. To structure asynchronous elements of ceremonies or meetings we were directly inspired by Florian Haas' presentation ['No, we won't have a video call for that'](https://xahteiwi.eu/resources/presentations/no-we-wont-have-a-video-call-for-that/).
+
+It is also important to understand that this is a multifaceted project, several strands of work (or even full projects) are simultaneously carried out and have their own ways of work and tracking tools. Therefore the Ways of Work and tools developed here focus on coordination among those with minimal added effort and avoiding interfering with day-to-day work.
+
+## Ways of work principles
+
+Ways of work is not only about reporting.
```

Comment:
Sometimes you use Ways of Work, sometimes Ways of work, and sometimes ways of work. 

If it's Ways of Work here, `is` is fine, but `Ways of work` now feels plural, so should be `are`.

Reply:
yeah, I have not been v consistent there... does it feel like Ways of Work could be a name? or is actually to generic to deserve it and ways of work would be better?
Is there anything else we would like to call it? like DSH's Ways of Work for example. It sometimes feels like we are talking about OUR very specific and particular Ways of Work, and others just talking about ways of work

Reply:
I definitely feel like we are talking about a specific Ways of Work thing here, rather than a generic one.